queries.sql

```diff
@@ -2,18 +2,22 @@
 
 SELECT * from animals WHERE name LIKE '%mon';
 
-SELECT * FROM animals WHERE To_date BETWEEN '2016-01-01' AND '2019-01-01';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
 
-SELECT * FROM animals WHERE neutered = 1 AND escape_attempts < 3;
+SELECT * FROM animals WHERE neutered = 'true' AND escape_attempts < 3;
 
 SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 
 SELECT name, escape_attempts FROM animals WHERE weight_kg >= 10.5;
 
-SELECT * FROM animals WHERE neutered = 1;
+SELECT * FROM animals WHERE neutered = 'true';
 
 SELECT * FROM animals WHERE name != 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
 
+UPDATE animals SET species = 'unspecified';
 
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
```

Comment:
- [ ] nice job mate, but there are some missing steps you didn't add, like updating the animals table by setting it to pokemon.
![Capture web_6-7-2022_153551_](https://user-images.githubusercontent.com/61562626/177577211-23146922-96f8-453c-97fa-fb571fe65aa1.jpeg)

- [ ] also you didn't take screenshot after rollback.
![Capture web_6-7-2022_154039_](https://user-images.githubusercontent.com/61562626/177577288-503eddfd-5b7e-41a7-977f-aae00566c934.jpeg)

![Capture web_6-7-2022_153551_](https://user-images.githubusercontent.com/61562626/177577370-54ad2ae9-afd7-486d-8dd0-49156662215c.jpeg)